docs/source/tutorial/understanding_evaluation.rst

```diff
@@ -180,6 +180,8 @@ may be additional known triples :math:`(h, r, t')` for :math:`t \neq t'`. If the
 filtered evaluation setting ignores for a given triple :math:`(h, r, t)` the scores of all other *known* true triples
 :math:`(h, r, t')`.
 
+TODO should the new content go here instead of in the reference section?
```

Comment:
@mali-git there's already a section on filtering in the tutorial section of the docs. I'm not sure if you were aware of it, but it also addresses the same subject that you wrote about in `src/pykeen/evaluation/__init__.py` at about the same granularity. I think it makes sense to move those docs in here and merge them, then to give a high-level example using the pipeline about how this stuff works. The remaining specific code belongs in the reference via `src/pykeen/evaluation/__init__.py` or maybe even directly in the class docs for Evaluator